doc/source/parameters/cfd/multiphysics.rst

```diff
@@ -5,21 +5,53 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
+
+    # Multiphase flow
+    set VOF 			= false
+    set interface sharpening 	= false
+    set conservation monitoring = false
+    set fluid index 		= 0
   end
 
 
-* The ``heat transfer`` parameter adds the heat transfer auxiliary physics. This is an advection-diffusion equation. If  ``viscous dissipation`` is set to true, viscous dissipation is taken into account in the heat transfer equation.
+* ``heat transfer``: controls if the heat transfer auxiliary physics are solved. This is an advection-diffusion equation. 
 
-* The ``buoyancy force`` parameter adds the buoyancy force in the Navier-Stokes equations. The buoyancy force is calculated using the Boussinesq approximation. Note that ``heat transfer`` is a prerequisite for ``buoyancy force``, and it must be set to true when we set ``buoyancy force`` to true.
+  When ``set heat transfer = true``, these optional parameters can be used:
+   * ``viscous dissipation``: controls if the viscous dissipation is taken into account in the heat transfer equation.
+   * ``buoyancy force``: controls if the buoyancy force is taken into account in the Navier-Stokes equations. The buoyancy force is calculated using the Boussinesq approximation.
+
+.. seealso::
+
+  The heat transfer solver is used in the example :doc:`../../examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid`.
 
 * The ``tracer`` parameter adds a passive tracer auxiliary physics. This is an advection-diffusion equation.
 
-* The ``VOF`` parameter enables defining a multiphasic simulation, with 2 fluids separated by a free surface. Volume-of-Fluid method is used, with the phase parameter following an advection equation. See :doc:`initial_conditions` for the definition of the VOF conditions and :doc:`physical_properties` for the definition of the physical properties of both fluids. At the moment, a maximum of two fluids is supported.
+.. seealso::
+
+  The tracer solver is used in the example :doc:`../../examples/multiphysics/tracer-through-cad-junction/tracer-through-cad-junction`.
+
+* ``VOF``: enables multiphase flow simulations, with two fluids separated by a free surface. Volume-of-Fluid method is used, with the phase parameter following an advection equation. 
+
+  See :doc:`initial_conditions` for the definition of the VOF conditions and `Physical properties - two phase simulations <https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html#two-phase-simulations>`_ for the definition of the physical properties of both fluids.
+
+  When ``set VOF = true``, these optional parameters can be used:
+    * ``interface sharpening``: controls if the interface sharpening method is used. Additional parameters can be found at :doc:`interface_sharpening`.
+    * ``conservation monitoring``: controls if conservation is monitored at each iteration, through the volume computation of the fluid with the given ``fluid index``. Results are outputed in a data table (`free_surface_monitoring.dat`).
```

Comment:
Maybe you can fix the issue with fluid index in this PR? :)

Reply:
This was a doc PR, so I would rather not. Let me investigate it and I will make a quick fix!